app/models/user.rb

```diff
@@ -163,6 +166,8 @@ class User < ApplicationRecord
   scope :boarding_buddy,          -> (status = true) { where(boarding_buddies: status)}
   scope :social_media,            -> (status = true) { where(social_media_manager: status)}
   scope :graphic_designer,        -> (status = true) { where(graphic_design: status)}
+  scope :active_volunteer,        -> (status = true) { where(active: status)}
```

Comment:
Maybe I don't know what's going on, but why use `status = true`? Could this be `where(active: true)`?

Reply:
Probably, I just haven't figured out the right way to do it.  Stripping it down to just that throws errors about having too many arguments.